Rework quoting and spelling

Benjamin-Piper opened the pull request:
Eberban is phonetic, but borrowed words are not.

In light of this, cen and skan have been removed to make
way for with a more nuanced system.

We now distinguish between denoting words, spelling,
sound, and quotes of words.

Starting from the most fundamental we have:

- ecima: the word for a word
- ecema: the word for a letter/grapheme
- eceuma: the word for a sound/phoneme
- ecama: a gramatically correct Eberban string
- ecoma: a string of data

These can be quoted using:

- ci: quote the next word

&nbsp;

- ce: quote a string/list of letters
- ceu: quote a string/list of sounds
- both of these are terminated by cei
 
 &nbsp;
 
- ca: a grammatically correct Eberban quote. Terminated by cai.

&nbsp;

- co: a quote of arbitrary length
- quote format: co \<delimiter>'\<quote content>'\<delimiter>
where \<delimiter> is a native word form

&nbsp;

- coi: a foreign quote of arbitrary length (skipped by speaker)

Spelling and uttering are referenced with:

- ei ce bu: E is spelled as A
- ei ceu bu: E is uttered as A

Enabling us to ask questions such as:

- poi gi be ci umia eicebu ba
- pa gi ce u mi i a cei

Benjamin-Piper committed:
Rework quoting and spelling

Eberban is phonetic, but borrowed words are not.

In light of this, cen and skan have been removed to make
way for with a more nuanced system.

We now distinguish between denoting words, spelling,
sound, and quotes of words.

Starting from the most fundamental we have:

- ecima: the word for a word
- ecema: the word for a letter/grapheme
- eceuma: the word for a sound/phoneme
- ecama: a gramatically correct Eberban string
- ecoma: a string of data

These can be quoted using:

- ci: quote the next word

- ce: quote a string/list of letters
- ceu: quote a string/list of sounds
- both of these are terminated by cei

- ca: a grammatically correct Eberban quote. Terminated by cai.

- co: a quote of arbitrary length
- quote format: co <delimiter>'<quote content>'<delimiter>
where <delimiter> is a native word form

- coi: a foreign quote of arbitrary length (skipped by speaker)

Spelling and uttering are referenced with:

- ei ce bu: E is spelled as A
- ei ceu bu: E is uttered as A

Enabling us to ask questions such as:

- poi gi be ci umia eicebu ba
- pa gi ce u mi i a cei

mia-entropy commented on dictionary/en.yaml:
```diff
@@ -1035,20 +1035,25 @@ cai:
   id: qd380clfxd
   family: "CI"
-  short: "Quote next word. [E:tca man] is word [word]."
+  short: "Quote next word. [E:tca man] is [word]."
```
```suggestion
  short: "Quote next word. [E:tca ecima] is the word [word]."
```

mia-entropy commented on dictionary/en.yaml:
```diff
@@ -1057,12 +1062,12 @@ cei:
   id: upxio5nkc0
   family: "CO"
-  short: "[E:tca skan] is a quote of an arbitrary string."
+  short: "[E:blu ecoma] is a quote of an arbitrary string."
```
```suggestion
  short: "[E:tca ecoma] is a quote of an arbitrary string."
```

mia-entropy commented on dictionary/en.yaml:
```diff
@@ -2141,7 +2133,7 @@ spi:
   family: "R"
   gloss: "says"
-  short: "[E:tce pan] says [A:tce* skan] to [O:tce* ma]."
+  short: "[E:tce pan] says [A:tce* ecama] to [O:tce* ma]."
```
```suggestion
  short: "[E:tce pan] says [A:tce* man] to [O:tce* ma]."
```
They could say things in other languages, thus I think it's better to have "man" for now.

Benjamin-Piper committed:
Apply Mia's tweaks from code review

For cna: E says A to O, the A argument is of type `tce* man` as they can
say things in other languages.

Co-authored-by: mia <63742169+mia-entropy@users.noreply.github.com>
